l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -390,8 +407,19 @@ def generate_supplier_self_invoice(self):
         supplier_invoice.partner_id = rc_type.partner_id.id
         supplier_invoice.journal_id = rc_type.supplier_journal_id.id
         for inv_line in supplier_invoice.invoice_line_ids:
-            inv_line.invoice_line_tax_ids = [
-                (6, 0, [rc_type.tax_ids[0].purchase_tax_id.id])]
+            line_tax_ids = inv_line.invoice_line_tax_ids
+            tax_ids = list()
+            for tax_mapping in rc_type.tax_ids:
+                for line_tax_id in line_tax_ids:
+                    if tax_mapping.original_purchase_tax_id == line_tax_id:
+                        tax_ids.append(tax_mapping.purchase_tax_id.id)
```

Comment:
Se un mapping avesse due righe con la stessa `original_purchase_tax_id` e quell'imposta fosse in una riga fattura, allora nella riga fattura verrebbero incluse tutte le imposte _mappate_, questo non poteva succedere con l'implementazione precedente a queste modifiche.
È davvero il comportamento voluto o basterebbe trovarne una e uscire? Un'altra soluzione sarebbe impedire che i mapping abbiano più righe con la stessa `original_purchase_tax_id`.